### PR TITLE
[ci-skip] [meta] Update Contributing.md to reflect new naming

### PR DESCRIPTION
**Pull request by TwoLeggedCat**
Also fixes somewhere where "Paper" should say "Purpur," so please fix that if you close this PR without merging :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -204,3 +204,6 @@ Again, notice that the field is always public, but the setter is always private.
 ```java
 return this.world.purpurConfig.useInhabitedTime ? this.w : 0;
 ```
+
+### Naming your Commits/PRs
+We are trying to use a new naming system where all (or most) commits pushed to master are prefixed with a tag like `[feat]`, `[fix]`, `[meta]`, etc and would appreciate if this were done in PR titles as well. While this is by no means a requirement, it allows users to see what's in a new build without reading the commit or even the entire message, and makes it easier to find a specific commit as well.
```
Comment (BillyGalbreath) [inline CONTRIBUTING.md]: Commits aren't that important, imho. It's the patch files/headers that are important.
```suggestion
We are trying to use a new naming system where all (or most) patches are prefixed with a tag like `[feat]`, `[fix]`, `[meta]`, etc and would appreciate if this were done in PR titles as well. While this is by no means a requirement, it allows users to see what's in a new build without reading the commit or even the entire message, and makes it easier to find a specific commit as well.
```

**Review by BillyGalbreath — CHANGES_REQUESTED** [review]

**Comment by cittyinthecloud** [comment]
I'd avoid naming patches with brackets if it makes the file names weird, iirc NTFS doesn't like them.

**Comment by katekerllenevich** [comment]
it wont affect the file names, git patching omits special characters for the file name

File: CONTRIBUTING.md (hunk 1 above)
Comment (TwoLeggedCat) [reply CONTRIBUTING.md]: As long as the merger renames the squashed commit appropriately so it shows on the downloads page; will change

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -204,3 +204,6 @@ Again, notice that the field is always public, but the setter is always private.
 ```java
 return this.world.purpurConfig.useInhabitedTime ? this.w : 0;
 ```
+
+### Naming your Commits/PRs
+We are trying to use a new naming system where all (or most) patches are prefixed with a tag like `[feat]`, `[fix]`, `[meta]`, etc and would appreciate if this were done in PR titles as well. While this is by no means a requirement, it allows users to see what's in a new patch without reading the file or even the entire message, and makes it easier to find a specific patches as well.
```
Comment (granny) [inline CONTRIBUTING.md]:
```suggestion
### Naming your Patches/PRs
We use a naming system with a similar concept to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).

Patches are prefixed with a type (`[feat]`, `[fix]`, `[meta]`, `[refactor]`, `[perf]`). The type should also appear in your PR title. 

This lets us and users:
* know what type of PR it is
* know what the patches do at a glance
* categorize the patches
```

Comment (TwoLeggedCat) [reply CONTRIBUTING.md]: "This lets us" -> "This lets us and our users?"

Comment (granny) [reply CONTRIBUTING.md]: edited

**Comment by granny** [comment]
This should be merged once all the patches follow the naming system.

**Review by granny — APPROVED** [review]

**Review by Oharass — APPROVED** [review]
LGTM

**Comment by granny** [comment]
merged into #351

**Pull request closed without merging**